projects/packages/publicize/src/rest-api/class-connections-controller.php

```diff
@@ -234,11 +260,17 @@ public static function get_connections( $run_tests = false ) {
 		}
 
 		$path = add_query_arg(
-			array( 'test_connections' => $run_tests ),
+			$args,
 			sprintf( '/sites/%d/publicize/connections', $site_id )
 		);
 
-		$response = Client::wpcom_json_api_request_as_user( $path, 'v2', array( 'method' => 'GET' ) );
+		$include = $args['include'] ?? '';
+
+		$blog_or_user = 'site-wide' === $include ? 'blog' : 'user';
```

Comment:
What's stopping a user adding the `site-wide=blog` param and getting all the connections? That may not be a problem if they're a trusted admin.

Reply:
We don't consume the `site-wide` param anywhere, do we?

Reply:
Sorry, got the parameter name wrong. I mean adding the `include=site-wide` parameter and getting the site's connections with user based authentication.

Reply:
I have removed the scope param from the API args